app/controllers/reservations_controller.rb

```diff
@@ -0,0 +1,43 @@
+class ReservationsController < ApplicationController
+  def index
+    @user = User.where(id: params[:user_id]).first
```

Comment:
- [ ] Using `find` here should be a better option then `where`, since you already have the `id`. This way you won't even need to use `first`.